### PR TITLE
Remove the logic responsible for produce kube-apiserver's audit config from configure-helper.

### PR DESCRIPTION
**Pull request by immutableT**
**What type of PR is this?**

> /kind cleanup

**What this PR does / why we need it**:
Generation of audit configuration for kube-apiserver is now handled in GKE.
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: cluster/gce/gci/configure-helper.sh [hunk 1]
```diff
@@ -921,168 +921,6 @@ EOF
   fi
 }
 
-# Write the config for the audit policy.
```
Comment (wojtek-t) [inline cluster/gce/gci/configure-helper.sh]: I understand your motivation of removing it from here.

That said, the fact that it lives here means that we test that as part of OSS scalability tests (and audit has non-negligible impact on performance/scalability). I can see a lot of value in its continuous testing at scale.

If we want to get rid of that, I would like to see some kind-of replacement that will mitigate the probability of regression that will appear to be hard to debug/fix once we observe it much later in GKE.

/hold

Comment (immutableT) [reply cluster/gce/gci/configure-helper.sh]: How are scalability tests initiated?

Comment (wojtek-t) [reply cluster/gce/gci/configure-helper.sh]: The cluster is created the same way as for all e2e tests (via prow)
Example config is here: https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-scalability/sig-scalability-release-blocking-jobs.yaml#L3

Comment (immutableT) [reply cluster/gce/gci/configure-helper.sh]: Thanks for explaining.

Comment (immutableT) [reply cluster/gce/gci/configure-helper.sh]: Thanks for explaining.

**Pull request closed without merging**